cli/cmd/plugin/standalone-cluster/log/log.go

```diff
@@ -89,11 +91,26 @@ type Logger interface {
 	Errorf(message string, args ...interface{})
 	// Progressf takes a progress counter, format string, arguments, and prints it as a standard log message.
 	// The progress counter will render as a quantity of "." characters defined by the value of count.
-	// Usage of Progressf typically involves a for loop feeding in a series of numbers such as
+	// Usage of Progressf typically involves calling either AnimateProgressf, AnimateStatusf,
+	// or writing a custom for loop feeding in a series of numbers such as
 	// 1,2,3,1,2,3,1,2,{exit due to condition}.
-	// Progressf will start each message with '\r', which will overwrite the last line. This gives the appearance of
-	// updating.
+	// Progressf will start each message with '\r', which will overwrite the last line.
+	// This gives the appearance of updating.
 	Progressf(count int, message string, args ...interface{})
```

Comment:
Is there a way we can make progressf private or something? Worried about this interface growing too large